src/AltinnCore/Common/Services/Implementation/FormSILocalDev.cs

```diff
@@ -121,45 +120,6 @@ public object GetPrefill(string org, string service, Type type, int partyId, str
             }
         }
 
```

Comment:
Why is this removed?

Reply:
it's not removed, its there in my branch as well